pkgs/data/documentation/mustache/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "mustache-${version}";
+  version = "1.0.2";
+
+  outputs = [ "out" ];
```

Comment:
This does not work, since there will be nothing left in `out`. I can do it manually, though.